### PR TITLE
Changes to accept non int ProviderParameters and to read all sensors …

### PR DESCRIPTION
**Pull request by bdagnino**
…in EPTS data

### CODE REVIEW

File: kloppy/infra/serializers/tracking/epts/metadata.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def _load_provider_parameters(parent_elm, value_mapper=None) -> Dict:
 
 def _load_periods(global_config_elm, frame_rate: int) -> List[Period]:
     provider_params = _load_provider_parameters(
-        global_config_elm.find("ProviderGlobalParameters"), value_mapper=int
```
Comment (bdagnino) [inline kloppy/infra/serializers/tracking/epts/metadata.py:43]: This was not needed as the result is the same whether you map to int or not, and it made the serialization crashed when there where provider parameters present that where not numbers.

File: kloppy/infra/serializers/tracking/epts/serializer.py [hunk 2]
```diff
@@ -120,8 +120,8 @@ def deserialize(
                     raw_data=inputs["raw_data"],
                     metadata=metadata,
                     sensor_ids=[
-                        "position"
-                    ],  # we don't care about other sensors
+                        sensor.sensor_id for sensor in metadata.sensors
```
Comment (bdagnino) [inline kloppy/infra/serializers/tracking/epts/serializer.py:123]: I believe we should load all sensors in the data, and the user can later filter out the ones they don't want.

Comment (koenvo) [reply kloppy/infra/serializers/tracking/epts/serializer.py:123]: That sounds like a good addition! The data from the other sensors is not loaded into the frames: https://github.com/PySport/kloppy/blob/master/kloppy/infra/serializers/tracking/epts/serializer.py#L42-L59

We would need a way to store additional information per frame.

File: kloppy/infra/serializers/tracking/epts/metadata.py (hunk 1 above)
Comment (koenvo) [reply kloppy/infra/serializers/tracking/epts/metadata.py:43]: Make sense!

**Pull request merged**